### ISSUE TITLE
jsonDecode failure

### ISSUE THREAD

**Issue by nihk** [urbanairship/airship-flutter#23]
I cloned your example project and threw in my Airship keys/required files/dependencies to try out push notifications on a Flutter app on a Pixel 3XL running Android 10. The notification gets delivered but doesn't seem to get parsed correctly by the Airship dart library. I put a breakpoint on the `Airship.onPushReceived.map` lambda and the `value` is always an empty string in my testing, which when passed to the `jsonDecode` method throws an exception. There's probably something wrong with my configuration somewhere, but I don't really know what that'd be. Any insight into why the `value` is always an empty string?

```
E/flutter ( 4702): [ERROR:flutter/lib/ui/ui_dart_state.cc(157)] Unhandled Exception: FormatException: Unexpected end of input (at character 1)
E/flutter ( 4702): 
E/flutter ( 4702): ^
E/flutter ( 4702): 
E/flutter ( 4702): #0      _ChunkedJsonParser.fail (dart:convert-patch/convert_patch.dart:1394:5)
E/flutter ( 4702): #1      _ChunkedJsonParser.close (dart:convert-patch/convert_patch.dart:512:7)
E/flutter ( 4702): #2      _parseJson (dart:convert-patch/convert_patch.dart:32:10)
E/flutter ( 4702): #3      JsonDecoder.convert (dart:convert/json.dart:495:36)
E/flutter ( 4702): #4      JsonCodec.decode (dart:convert/json.dart:153:41)
E/flutter ( 4702): #5      jsonDecode (dart:convert/json.dart:96:10)
E/flutter ( 4702): #6      Airship.onPushReceived.<anonymous closure> (package:airship/src/airship.dart:257:61)
E/flutter ( 4702): #7      _MapStream._handleData (dart:async/stream_pipe.dart:229:21)
E/flutter ( 4702): #8      _ForwardingStreamSubscription._handleData (dart:async/stream_pipe.dart:166:13)
E/flutter ( 4702): #9      _rootRunUnary (dart:async/zone.dart:1134:38)
E/flutter ( 4702): #10     _CustomZone.runUnary (dart:async/zone.dart:1031:19)
E/flutter ( 4702): #11     _CustomZone.runUnaryGuarded (dart:async/zone.dart:933:7)
E/flutter ( 4702): #12     _BufferingStreamSubscription._sendData (dart:async/stream_impl.dart:338:11)
E/flutter ( 4702): #13     _DelayedData.perform (dart:async/stream_impl.dart:593:14)
E/flutter ( 4702): #14     _StreamImplEvents.handleNext (dart:async/stream_impl.dart:709:11)
E/flutter ( 4702): #15     _PendingEvents.schedule.<anonymous closure> (dart:async/stream_impl.dart:669:7)
E/flutter ( 4702): #16     _rootRun (dart:async/zone.dart:1122:38)
E/flutter ( 4702): #17     _CustomZone.run (dart:async/zone.dart:1023:19)
E/flutter ( 4702): #18     _CustomZone.runGuarded (dart:async/zone.dart:925:7)
E/flutter ( 4702): #19     _CustomZone.bindCallbackGuarded.<anonymous closure> (dart:async/zone.dart:965:23)
E/flutter ( 4702): #20     _rootRun (dart:async/zone.dart:1126:13)
E/flutter ( 4702): #21     _CustomZone.run (dart:async/zone.dart:1023:19)
E/flutter ( 4702): #22     _CustomZone.runGuarded (dart:async/zone.dart:925:7)
E/flutter ( 4702): #23     _CustomZone.bindCallbackGuarded.<anonymous closure> (dart:async/zone.dart:965:23)
E/flutter ( 4702): #24     _microtaskLoop (dart:async/schedule_microtask.dart:43:21)
E/flutter ( 4702): #25     _startMicrotaskLoop (dart:async/schedule_microtask.dart:52:5)
E/flutter ( 4702): 
```

**Comment by DebugTheCode** [comment]
I am having the same problem just now. Also used the example code from the repo.

**Comment by rlepinski** [comment]
Thanks, we will take a look into this today

**Comment by rlepinski** [comment]
Found the issue, will have a fix on Monday

**Comment by DebugTheCode** [comment]
Cool, thank you! Have a great weekend.

**Comment by rlepinski** [comment]
Fixed in 1.0.1
